transport-parent/transport-netty/src/main/java/io/scalecube/transport/netty/tcp/TcpReceiver.java

```diff
@@ -42,9 +41,7 @@ private TcpServer newTcpServer(ReceiverContext context) {
         .option(ChannelOption.TCP_NODELAY, true)
         .option(ChannelOption.SO_KEEPALIVE, true)
         .option(ChannelOption.SO_REUSEADDR, true)
-        .bootstrap(
-            b ->
-                BootstrapHandlers.updateConfiguration(
-                    b, "inbound", new TcpChannelInitializer(config.maxFrameLength())));
+        .doOnChannelInit((connectionObserver, channel, remoteAddress) -> 
+            new TcpChannelInitializer(config.maxFrameLength()).accept(connectionObserver, channel));
```

Comment:
Seems it's not enough (test are not passed), so need to reverse ordering of adding handlers in `TcpChannelInitializer`:

```java
    pipeline.addFirst(new ExceptionHandler());
    pipeline.addFirst(
        new LengthFieldBasedFrameDecoder(
            maxFrameLength, 0, LENGTH_FIELD_LENGTH, 0, LENGTH_FIELD_LENGTH));
    pipeline.addFirst(new LengthFieldPrepender(LENGTH_FIELD_LENGTH));
```